files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/vue_methods_events_models/index.md

```diff
@@ -43,7 +43,7 @@ slug: >-
 
 我们的 app 可以展示待办事项列表，但是我们不能更新该列表，除非手动更改代码。让我们新建一个组件来允许我们添加新的待办项。
 
-在 components 目录下，新建文件 `ToDoForm.vue`.
+在 components 目录下，新建文件 `ToDoForm.vue`。
```

Comment:
could you add this to ordered list:

![image](https://github.com/mdn/translated-content/assets/15844309/21a832ab-4e66-45f7-8c1d-7d85436750ce)